tools/src/wyvern/tools/tests/EffectSystemTests.java

```diff
@@ -384,9 +384,85 @@ public void testNetwork19() throws ParseException {
         TestUtil.doTestScriptModularly(PATH, "effects.testNetwork19", Util.stringType(), new StringLiteral("Network19 with effects"));
     }
 
+    @Test
```

Comment:
Following the format for the rest of the tests in this test suite, please add comments explaining what each test checks.